.travis.yml

```diff
@@ -22,19 +22,6 @@ script:
   - coverage run systers_portal/manage.py test --settings=systers_portal.settings.testing
   - flake8 systers_portal
 notifications:
-  irc:
-    channels: "irc.freenode.org#systers-dev"
-    template:
-      - "%{repository}@%{branch}: %{message} (%{build_url})"
-    on_success: change
-    on_failure: change
-    use_notice: true
-  email: false
-  webhooks:
-    urls:
-      - https://webhooks.gitter.im/e/c403378d624b2391961e
-    on_success: change
-    on_failure: always
-    on_start: false
+  slack: '<account>:<token>#portal'
```

Comment:
Can we set it in travis environment variables and use it here?

Reply:
Yes we can good suggestion...

Reply:
@Fenn-CS Did you update this?

Reply:
@ExactlyMay  this is issue is pending for admin approval as you specified.

Reply:
@ExactlyMay everything is now sorted out, I am just waiting for slack integration details t complete this. :)

Reply:
@Fenn-CS did you incorporate the change suggested by Monal here ??

Reply:
@MehaKaushik that would be included with the token and account details.